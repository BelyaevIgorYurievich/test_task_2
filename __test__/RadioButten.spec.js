import React from 'react';
import { shallow } from 'enzyme'

import RadioButton from '../source/radio_button/index.js'

describe('RadioButton', () => {
    it('отображается корректно', () => {
        const renderedValue = shallow(<RadioButton />);
        
        expect(renderedValue).toMatchSnapshot();
    });
})