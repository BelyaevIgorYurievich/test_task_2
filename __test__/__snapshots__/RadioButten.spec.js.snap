// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton отображается корректно 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RadioButton
    isActive={false}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        false,
        <span
          className="field-name"
        />,
      ],
      "className": "radio-butten",
      "onClick": undefined,
    },
    "ref": null,
    "rendered": Array [
      false,
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "field-name",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          false,
          <span
            className="field-name"
          />,
        ],
        "className": "radio-butten",
        "onClick": undefined,
      },
      "ref": null,
      "rendered": Array [
        false,
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "field-name",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
